specification/sql/resource-manager/Microsoft.Sql/preview/2015-05-01-preview/examples/ManagedInstanceListByResourceGroup.json

```diff
@@ -47,7 +49,8 @@
                         "storageSizeInGB": 1024,
                         "licenseType": "Full",
                         "collation":  "SQL_Latin1_General_CP1_CI_AS",
-                        "dnsZone": "1b4e2caff2530"
+                        "dnsZone": "1b4e2caff2530",
+                        "publicDataEndpointEnabled" :  "false"
```

Comment:
Is proxyOverride intentionally missing?

Reply:
This is the case when the value for that column is null (instance created before introducing new property).